unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/GotoPanelHUD/GotoPanelFeature.cs

```diff
@@ -0,0 +1,16 @@
+public class GotoPanelFeature : IPlugin
+{
+    public GotoPanelHUDController gotoPanelHUDController;
+
+    public GotoPanelFeature()
+    {
+        gotoPanelHUDController = CreateController();
+    }
+
+    internal virtual GotoPanelHUDController CreateController() => new GotoPanelHUDController();
```

Comment:
Is this useful? I believe the intention is to mock the controller, but the plugin has no business logic to test.